code_style/README.md

```diff
@@ -11,7 +11,7 @@ WARNING this will overwrite your current ``~/.eslintrc.json` if you have one.
 
 `ln -s THIS_REPO/code_style/dot-eslintrc.json ~/.eslintrc.json`
 
-To install eslint: `npm install -g eslint eslint-plugin-react`
+To install eslint: `npm install -g eslint eslint-plugin-meteor eslint-plugin-react`
```

Comment:
Meteor recommends the base airbnb config. Can we use the recommended install from the guides ?
`meteor npm install --save-dev eslint eslint-plugin-react eslint-plugin-meteor eslint-config-airbnb
`